gemhardware/devices/src/common/optohybrid/HwOptoHybrid.cc

```diff
@@ -228,19 +228,13 @@ void gem::hw::optohybrid::HwOptoHybrid::configureVFATs(std::map<std::string, uin
   //} GEM_CATCH_RPC_ERROR("HwOptoHybrid::configureVFATs", gem::hw::devices::exception::Exception);
 }
 
-
-void gem::hw::optohybrid::HwOptoHybrid::configureGBT(uint8_t const& gbtID, uint32_t const* gbtcfg)
+void gem::hw::optohybrid::HwOptoHybrid::configureGBT(uint8_t const& gbtID, std::array<const uint8_t, ::gbt::CONFIG_SIZE> const& gbtcfg)
 {
-  gbt::config_t t_gbtconfig = {*(uint8_t*) gbtcfg};
+  configureGBT(gbtID, gbtcfg.data());
   try {
     xhal::common::rpc::call<::gbt::writeGBTConfig>(rpc, static_cast<uint32_t>(m_link), static_cast<uint32_t>(gbtID), t_gbtconfig);
```

Comment:
Does not compile: `configureGBT(gbtID, gbtcfg.data());` is not type compliant, would create an inifinte loop and is useless. `t_gbtconfig`, on line 235, is no longer a variable, replaced by `gbtcfg`.

Reply:
ooups, missed the change in the commit. My bad, sorry. Fixed now